src/cryptoadvance/specter/util/reflection.py

```diff
@@ -78,7 +78,7 @@ def get_classlist_of_type_clazz_from_modulelist(clazz, modulelist):
             module = import_module(fq_module_name)
         except ModuleNotFoundError as e:
             # ToDo: make it somehow clear where specific extensions are coming from: external or within same repo
-            logger.warn(
+            raise SpecterError(
```

Comment:
Please revert this change. This additional raising of an exception is triggered every time for me 

```
ModuleNotFoundError: No module named 'cryptoadvance.specterext.liquidissuer'

During handling of the above exception, another exception occurred:

```

Reply:
Why is that the case for you? This doesn't happen on my computer. In the `requirements.in`, you can read:
```
# Extensions
cryptoadvance-liquidissuer==0.2.4
specterext-exfund==0.1.7
specterext-faucet==0.1.2
```
So this should not happen. `continue` here is imho hiding configuration issues exactly like the one you currently have?!.

Reply:
I agree that your change reveals previously existing problems. 

However I just did `pip install --require-hashes -r requirements.txt`  and still get this error when I run 
```
python3 setup.py install && python3 -m cryptoadvance.specter server --config DevelopmentConfig     --port 1234   --ssl --debug
```

Is there anything else needed for these services?